File: Arrays/Easy/1491/1491.py
from typing import List

# class Solution:
#     def average(self, salary: List[int]) -> float:
#         salary.remove(min(salary))
#         salary.remove(max(salary))
#         return sum(salary) / len(salary)

class Solution:
    def average(self, salary: List[int]) -> float:
        sum = 0
        max = salary[0]
        min = salary[0]
        for s in salary:
            if s > max:
                max = s
            elif s < min:
                min = s
            sum += s
        sum = sum - min - max
        return sum / (len(salary) - 2)

File: Arrays/Easy/1491/1491.c
double average(int* salary, int salarySize){
    double sum = 0.;
    double max = salary[0];
    double min = salary[0];
    for(int i = 0; i < salarySize; i++){
        if(salary[i] > max){
            max = salary[i];
        }else if(salary[i] < min){
            min = salary[i];
        }
        sum += salary[i];
    }
    sum = sum - min - max;
    return sum / (salarySize - 2);
}